go/cli/mcap/cmd/attachments.go

```diff
@@ -52,16 +61,7 @@ var attachmentsCmd = &cobra.Command{
 		}
 		filename := args[0]
 		err := utils.WithReader(ctx, filename, func(matched bool, rs io.ReadSeeker) error {
-			reader, err := mcap.NewReader(rs)
-			if err != nil {
-				return fmt.Errorf("failed to get reader: %w", err)
-			}
-			info, err := reader.Info()
-			if err != nil {
-				return fmt.Errorf("failed to get info: %w", err)
-			}
-			printChunks(os.Stdout, info.ChunkIndexes)
-			return nil
+			return printAttachments(os.Stdout, rs)
```

Comment:
fwiw I think the previous implementation would be correct if we had instead called,

```
printAttachments(os.Stdout, info.AttachmentIndexes)
```

Reply:
Agreed, except for the case where an mcap has no summary.